pygtc/pygtc.py

```diff
@@ -802,7 +808,7 @@ def _mpy(self, y):
                     prior1d = priors[i]
             # Plot!
             ax = __plot1d(ax, nChains, chainsForPlot1D, weights, nBins,
-                          smoothingKernel, filledPlots, colors,
+                          smoothingKernel, filledPlots, colors, linewidth,
```

Comment:
Shouldn't you be passing `contourLinewidth` here? I don't see where `linewidth` has been defined?